service_capacity_modeling/models/common.py

```diff
@@ -129,7 +129,7 @@ def network_services(
     # inter region. This is the number of regions minus 1
     inter_txfer = context.services.get("net.inter.region", None)
     if inter_txfer:
-        price_per_gib = inter_txfer.annual_cost_per_gib
+        price_per_gib = inter_txfer.annual_cost_per_gib[0][1]
```

Comment:
If we do the annual_cost then we'd just size the net.inter.region `Service` and then call annual_cost and multiply by number of regions